### PR TITLE
Print top 5 latency metrics even if they are below threshold

### PR DESCRIPTION
**Pull request by fgrzadkowski**
This will give us more insight how the latency changes over time (e.g. whether we had a large regression in this run, or it was close to threshold for some time).

### CODE REVIEW

File: test/e2e/util.go [hunk 1]
```diff
@@ -1073,16 +1082,28 @@ func HighLatencyRequests(c *client.Client, threshold time.Duration, ignoredResou
 	if err != nil {
 		return 0, err
 	}
+	sort.Sort(sort.Reverse(LatencyMetricByLatency(metrics)))
 	var badMetrics []LatencyMetric
+	top := 5
 	for _, metric := range metrics {
-		if !ignoredResources.Has(metric.resource) &&
-			metric.verb != "WATCHLIST" &&
+		if ignoredResources.Has(metric.Resource) || metric.Verb == "WATCHLIST" {
```
Comment (wojtek-t) [inline test/e2e/util.go]: FYI: This will need to be rebased once #8770 is merged.

**Comment by wojtek-t** [comment]
LGTM

File: test/e2e/util.go (hunk 1 above)
Comment (wojtek-t) [inline test/e2e/util.go]: #8770 is merged - please rebase :)

**Comment by fgrzadkowski** [comment]
Rebased.

**Pull request merged**